src/draw/shapes.typ

```diff
@@ -333,7 +335,22 @@
       default: "arc-start",
       name: name,
       offset-anchor: anchor,
-      transform: ctx.transform,
+      // rotate around arc-start by -style.rotation
+      transform: matrix.translate(
+        matrix.mul-mat(
+          matrix.translate(ctx.transform, (
+            +arc-start.at(0),
+            -arc-start.at(1),
```

Comment:
This is not correct. Transform points using `util.apply-transform` first, do not expect flipped y-direction.